sql/sql_yacc.yy

```diff
@@ -454,7 +454,7 @@ void warn_about_deprecated_binary(THD *thd)
   1. We do not accept any reduce/reduce conflicts
   2. We should not introduce new shift/reduce conflicts any more.
 */
-%expect 102
+%expect 124
```

Comment:
What are the causes for the new shift/reduce conflicts?

Reply:
Not sure about the reason at the moment. It looks like it's caused by new rule "ATTACH_SYM EXPLICIT_SYM ident_or_text SNAPSHOT_SYM" but I'm not sure how it could lead to conflicts as ATTACH_SYM is a new symbol and it isn't used in other rules.

Reply:
Thanks for following up on this. This was surprising to me too since the original change in 5.6 didn't increase the expected number of conflicts and, as you said, the command symbols are not used in other rules.

The hlc issue seems to be fixed. I'm re-running other tests.

Reply:
After a second look at this shift/reduce conflicts it looks like the reason for this is that ATTACH_SYM symbol gets matched as a candidate for some already existent rules. For example

State 4889
  2913 sp_tail: PROCEDURE_SYM sp_name $@67 '(' $@68 sp_pdparam_list ')' $@69 sp_c_chistics $@70 . sp_proc_stmt

    ATTACH_SYM                   shift, and go to state 619
    ATTACH_SYM  [reduce using rule 400 ($@14)]

The ATTACH_SYM is at the moment added into ident_keywords_unambiguous list. Adding it into ident_keywords_ambiguous_2_labels instead seem to fix conflicts.